devices.js

```diff
@@ -4577,13 +4577,13 @@ const devices = [
 
     // Blaupunkt
     {
-        zigbeeModel: ['SCM-R_00.00.03.15TC', 'SCM_00.00.03.14TC'],
+        zigbeeModel: ['SCM-2_00.00.03.15', 'SCM-R_00.00.03.15TC', 'SCM_00.00.03.14TC'],
         model: 'SCM-S1',
         vendor: 'Blaupunkt',
         description: 'Roller shutter',
         supports: 'open/close',
         fromZigbee: [fz.cover_position_via_brightness, fz.cover_state_via_onoff],
-        toZigbee: [tz.cover_position_via_brightness, tz.cover_open_close_via_brightness],
+        toZigbee: [tz.cover_position_via_brightness, tz.cover_open_close_via_brightness, tz.osram_cmds],
```

Comment:
Why was `osram_cmds` added here?

Reply:
Oops, that second commit should not be part of the pull request. I wanted to test around with this, because you can set the transition time to the roller shutter. By default it will keep each relais for four minutes, but you can set this to another value. It seems to me, that the command ist identical to the set-transition method in osram_cmds.